x-pack/legacy/plugins/task_manager/migrations.ts

```diff
@@ -12,5 +12,25 @@ export const migrations = {
       ...doc,
       updated_at: new Date().toISOString(),
     }),
+    '7.6.0': moveIntervalIntoSchedule,
```

Comment:
I think this is messed things up in my migration. You want `7.6.0` here?

Reply:
yup, should be 7.6.0, as this code will be released in that version, no?